src/core/Akka.Persistence.Tests/Serialization/MessageSerializerRemotingSpec.cs

```diff
@@ -202,44 +201,5 @@ public void MessageSerializer_should_custom_serialize_AtomicWrite_messages_durin
                 ExpectMsg("p.b.");
             });
         }
-
-        [Fact]
-        public void MessageSerializer_should_serialize_manifest_provided_by_EventAdapter()
-        {
-            var p1 = new Persistent(new MyPayload("a"), sender: TestActor).WithManifest("manifest");
-            var serializer = _serialization.FindSerializerFor(p1);
-            var bytes = serializer.ToBinary(p1);
-            var back = (Persistent)serializer.FromBinary(bytes, typeof (Persistent));
-
-            back.Manifest.ShouldBe(p1.Manifest);
-        }
-
-        [Fact]
-        public void MessageSerializer_should_serialize_state_change_event()
-        {
-            var p1 = new Persistent(new PersistentFSM.StateChangeEvent("a", TimeSpan.FromSeconds(10)), sender: TestActor);
-            var serializer = _serialization.FindSerializerFor(p1);
```

Comment:
We should not test all our serialization infrastructure here, just a concrete serializer.

Reply:
You moved this and the test above to a seperate spec file. But the tests remain the same. So what do you mean ?

Reply:
I stopped using `FindSerializerFor` and added a test for AtLeastOnceDeliverySnapshot
We have integrations tests for all standard persistence messages in Persistence.TCK